src/components/dialogmote/referat/Referat.tsx

```diff
@@ -261,7 +285,7 @@ const Referat = ({
 
   const debouncedAutoSave = useDebouncedCallback(
     (values: ReferatSkjemaValues) => {
-      if (!(ferdigstillDialogmote.isLoading || endreReferat.isLoading)) {
+      if (!isSendingReferat() && !isEmptyReferat(values)) {
```

Comment:
De Morgan strikes again? 🎓

Reply:
Jeg følte at det var mer naturlig å lese det sånn det er nå, siden det er to separate ting som sjekkes 🤔 
I stedet for: "ikke disse to tingene",
så har vi: "ikke den ene tingen, og ikke den andre tingen"